core/templates/dev/head/library/library.html

```diff
@@ -56,16 +56,16 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
                translate="I18N_LIBRARY_NO_EXPLORATION_GROUPS">
           </div>
 
-          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
+          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index" ng-init="groupIndex = $index">
```

Comment:
But wouldn't the additional code you add be before the tiles? If so, then $index should refer to the $index of the most closest for loop you're "inside", which in this case would be the group...